### PR TITLE
Delete duplicate logs for kubeadm upgrade versions

### PR DESCRIPTION
**Pull request by CaoDonghui123**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it:

[root@cdh-k8s-1 ~]# kubeadm upgrade plan
[upgrade/config] Making sure the configuration is correct:
[upgrade/config] Reading configuration from the cluster...
[upgrade/config] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -o yaml'
[preflight] Running pre-flight checks.
[upgrade] Running cluster health checks
[upgrade] Fetching available versions to upgrade to
[upgrade/versions] Cluster version: v1.20.2
[upgrade/versions] kubeadm version: v1.20.2
[upgrade/versions] Latest stable version: v1.20.3
[upgrade/versions] Latest stable version: v1.20.3
_**[upgrade/versions] Latest version in the v1.20 series: v1.20.3
[upgrade/versions] Latest version in the v1.20 series: v1.20.3**_

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/phases/upgrade/versiongetter.go [hunk 1]
```diff
@@ -135,9 +134,6 @@ func NewOfflineVersionGetter(versionGetter VersionGetter, version string) Versio
 func (o *OfflineVersionGetter) VersionFromCILabel(ciVersionLabel, description string) (string, *versionutil.Version, error) {
 	if o.version == "" {
 		versionStr, version, err := o.VersionGetter.VersionFromCILabel(ciVersionLabel, description)
-		if err == nil {
-			fmt.Printf("[upgrade/versions] Latest %s: %s\n", description, versionStr)
-		}
 		return versionStr, version, err
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/upgrade/versiongetter.go:137]: you can `return o.VersionGetter.VersionFromCILabel(ciVersionLabel, description)` directly here

please keep the commits squashed to one.

**Comment by pacoxu** [comment]
/retitle  Delete duplicate logs for kubeadm upgrade versions

**Comment by pacoxu** [comment]
> [upgrade/versions] Cluster version: v1.20.2
> [upgrade/versions] kubeadm version: v1.20.2
> [upgrade/versions] Latest stable version: v1.20.3
> [upgrade/versions] Latest stable version: v1.20.3
> _**[upgrade/versions] Latest version in the v1.20 series: v1.20.3**_ 
> _**[upgrade/versions] Latest version in the v1.20 series: v1.20.3**_

Are ` Latest stable version: v1.20.3` duplicated as well? 
Would you fix both?

**Comment by CaoDonghui123** [comment]
`Latest stable version: v1.20.3` has been fixed #98728

**Pull request merged**